actors/builtin/multisig/multisig_actor.go

```diff
@@ -149,7 +154,8 @@ func (a Actor) Propose(rt vmr.Runtime, params *ProposeParams) *ProposeReturn {
 	var st State
 	var txn *Transaction
 	rt.State().Transaction(&st, func() {
-		if !isSigner(rt.ResolveAddress, &st, callerAddr) {
+		proposerIsSigner := isSigner(callerAddr, st.Signers)
+		if !proposerIsSigner {
```

Comment:
Nit: I would fold these back to one line; the predicate code is already pretty clear. Possibly rename `callerAddr` for even more clarity.